scripts/changelog-check

```diff
@@ -2,25 +2,50 @@
 require 'open3'
 require 'optparse'
 
-CHANGELOG_REGEX = %r{^\+- (?<category>[\w -]{2,}): (?<change>.+) \((?<pr>(?:#\d+, ?)*(?:#\d+))\)$}
-# A valid diff line is in the form of:
-# +- CATEGORY: CHANGE_DESCRIPTION (#PULL_REQUEST_NUMBER)
-def is_valid_changelog_diff_addition?(line)
-  matches = CHANGELOG_REGEX.match(line)
-  !matches.nil?
+CHANGELOG_REGEX =
+  %r{^changelog: (?<category>[\w -/]{2,}), (?<subcategory>[A-Z][\w -]{2,}), (?<change>.+)$}
+CATEGORIES = [
+  'Improvements', 'Accessibility', 'Bug Fixes', 'Behind the Scenes Changes'
```

Comment:
I boldly edited our change categories down, definitely open to suggestions/veto on this

Reply:
I was looking at some other projects for precedent. I wonder if "Internal" could work for "Behind the Scenes Changes" ?

I don't really feel strongly if we'd rather keep to how we've had it though.

Reply:
I agree with that, will switch it!